src/HttpHeader.cc

```diff
@@ -736,6 +736,37 @@ HttpHeader::refreshMask()
     }
 }
 
+bool
+HttpHeader::tooLarge() const
+{
+    size_t max = 0;
+    switch (owner) {
+    case hoRequest:
+        max = Config.maxRequestHeaderSize;
+        break;
+    case hoReply:
+        max = Config.maxReplyHeaderSize;
+        break;
+#if USE_HTCP
+    case hoHtcpReply:
+        // HTCP messages have a fixed header length
```

Comment:
It is not clear why a fixed header length implies a negative tooLarge() answer. For _example_, what if that "fixed header length" exceeds Config.maxReplyHeaderSize?

P.S. I doubt that HTCP responses have a fixed HTTP header length -- that would not be practical. HTCP reply structure may impose some header length limit, but the actual received HTTP headers ought to vary in length.

Reply:
> I doubt that HTCP responses have a fixed HTTP header length

You are right, I meant that HTCP header itself has a fixed [length](https://datatracker.ietf.org/doc/html/rfc2756) (comprising LENGH/MAGOR/MINOR numbers). When creating an HTCP response, htcpTstReply() creates and packs an HTTP header. I think that there should be the same maxReplyHeaderSize limitation in this context. Note that HTCP messages are limited by 8192 limitation in htcpTstReply(), basically meaning that HTTP header in HTCP context is less than this value.